src/igvfd/schemas/donor.json

```diff
@@ -81,6 +81,20 @@
                 "linkTo": "PhenotypeTerm"
             }
         },
+        "phenotypic_feature": {
```

Comment:
should be plural after last week discussion
phenotypic_features

Reply:
Done